### DISCUSSION TITLE
THE BIGGEST APPWRITE PROBLEMS 🤔🤔🤔

### DISCUSSION THREAD

**Discussion by tananga** [appwrite/appwrite#3267] [General]
Hi. I decided to create a list of biggest AppWrite problems from my perspective as a frontend Developer who have 6 years experience in web/mobile development and made a few projects with Firebase.  

nr 1. AppWrite cloud:
This should be a priority number one. This would be helpful for beginners who wants to try AppWrite and they dont know exacly what Docker is and why it is sooo great. When I build something, I want to show it to my friends and I cant show them localhost. Sure I can create DigitalOcean or other remote instance but documentation is still very poor.

nr 2. No good tutorials on youtube:
This problem is even bigger than first one. I love the idea behind AppWrite but I just dont have a lot of resources to learn more about it. Documentation is good but still a lot of steps is missing. This is a huge mistake. You guys created a fantastic tool but without good tutorials no-one will use it. I found a channel on youtube named: React Bits (Damodar Lohani) and he is doing a great job but also a lots of steps are skipped or he is just too advanced in AppWrite to made his videos beginners friendly. I have the advanced equipment and the ability to record tutorials like that but I still dont understand few concepts. If you as a AppWrite team would like me to record  this tutorials I need someone from the team to have a one by one call with me and clarify things which are still not very clear for me. I am available whenever you want. 

Have a nice day
You are doing a great job.

**Comment by stnguyen90** [comment]
> nr 1. Appwrite cloud: This should be a priority number one

The core team is actively working to make [Appwrite Cloud](https://appwrite.io/cloud) ready ASAP.

> I just dont have a lot of resources to learn more about it

There are a ton of [resources](https://github.com/appwrite/awesome-appwrite) on Appwrite. The hardest part is keeping everything up to date, especially with how much things are still actively changing. However, I think the core team is actively trying to make sure the [30 Days of Appwrite](https://30days.appwrite.io/) series and the [Demos](https://appwrite.io/docs/getting-started-for-web#demos) are up to date. 

That said, the core team is working on [making more video tutorials](https://dev.to/appwrite/getting-started-with-appwrite-on-youtube-2mbm).

> React Bits (Damodar Lohani)

Damodar is actually part of the core team :wink:

**Reply by tananga** [reply]
@stnguyen90 There is a lot of resources but most of them are very chaotic and important steps are missing. I would like to create a series of youtube tutorials from 0 to hero with all the details and without skipping anything. The only thing which I need from you guys is to answer few questions which I still have and Im ready to go.

**Reply by eldadfux** [reply]
You can reach out to the team directly on Discord, we'd love to help with anything we can.

**Reply by CEbbinghaus** [reply]
I can feel the struggle with trying to keep all the docs up to date and everything synced. just looking at the [appwrite/lite](https://github.com/appwrite/lite) I see an amazing idea that had to take a backseat to the active development and never got updated. I agree that It can feel a little light in terms of resources out there especially on youtube however I think it's a matter of time and adoption. As more people start using appwrite for more projects more will be written about it and more videos will be made as a consequence. It's a self fuelling machine one just has to get started first. And a big part of that is just Quality and accessibility. Appwrite cloud is going to be the first big step as well as being a potential source of income allowing them to grow their team and resources. From there, more features and performance improvements are going to probably make a majority of the reasons it gains more traction.
